### PR TITLE
Read serial

### PR DESCRIPTION
**Pull request by SeptimiuVana**

### CODE REVIEW

File: sdk/src/cameras/itof-camera/camera_itof.cpp [hunk 1]
```diff
@@ -83,6 +83,7 @@ CameraItof::CameraItof(
     m_details.uBootVersion = ubootVersion;
     m_details.kernelVersion = kernelVersion;
     m_details.sdCardImageVersion = sdCardImageVersion;
+    m_details.serialNumber = "";
```
Comment (dNechita) [inline sdk/src/cameras/itof-camera/camera_itof.cpp:86]: If serialNumber is of type string, then the default constructor will create an empty string. No need to set it with an empty string.

File: sdk/src/cameras/itof-camera/camera_itof.cpp [hunk 2]
```diff
@@ -1190,27 +1193,31 @@ void CameraItof::freeConfigData(void) {
...
-        serialNumber = m_details.serialNumber;
-        if (serialNumber == "")
-            stat = aditof::Status::UNAVAILABLE;
-        else
-            stat = aditof::Status::OK;
+    using namespace aditof;
+    Status status = Status::OK;
+
+    if (useCacheValue) {
+        if (m_details.serialNumber != "") {
```
Comment (dNechita) [inline sdk/src/cameras/itof-camera/camera_itof.cpp:1200]: you can check is a string is empty or not. It think the method is called Emtpy()

File: sdk/src/cameras/itof-camera/camera_itof.cpp [hunk 3]
```diff
@@ -1190,27 +1193,31 @@ void CameraItof::freeConfigData(void) {
...
-}
 
-void CameraItof::setSerialNumber(uint8_t *ccbContent, int ccbFileSize) {
-    char buff[32];
-    // 176 = 0xB0
-    const char *data = reinterpret_cast<const char *>(ccbContent);
-    strncpy(buff, data + 176, 32);
-    m_details.serialNumber = std::string(buff);
-    LOG(INFO) << "m_details.serialNumber=" << m_details.serialNumber;
+    m_details.serialNumber = std::string((char *)serial, 32);
```
Comment (dNechita) [inline sdk/src/cameras/itof-camera/camera_itof.cpp:1217]: Please use a C++ cast

Comment (SeptimiuVana) [reply sdk/src/cameras/itof-camera/camera_itof.cpp]: Fixed.

**Review by dNechita — APPROVED** [review]

**Pull request merged**